### DISCUSSION TITLE
update coreui pro version

### DISCUSSION THREAD

**Discussion by fiqryq** [coreui/coreui#381] [Q&A] [answered]
how to update the version of core UI pro without downloading another version from web?

![photo_2022-03-16 11 56 01](https://user-images.githubusercontent.com/25787603/158519446-06ef19c8-8abd-4555-9472-4ef174479021.jpeg)

**Comment by mrholek** [comment, marked as answer]
@fiqryq please update `@coreui/coreui` to `@coreui/coreui-pro` in package.json, reinstall it `npm install`, and change `node_modules/@coreui/coreui/...` links to `node_modules/@coreui/coreui-pro/...`
